### DISCUSSION TITLE
posts/windows_hugo_github_pages_blog/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#2] [Announcements]
# posts/windows-hugo-blog-Github/

前提 默认本地已安装了 Git、VSCode。 hugo 配置 1. 安装 Hugo 推荐使用 Hugo extended 版本 预构建的二进制文件 访问最新版本页面，然后向下滚动到Assets部分。

https://haoyep.com/posts/windows-hugo-blog-Github/

**Comment by Dragonsca1es** [comment]
请问在搭建中遇到的一些问题博主可以提供些帮助建议吗？“
1.在archetypes目录中无posts.md
2.按步骤操作后在content\posts目录下创建测试网页开始没有默认模板
3.hugo server -D 会产生错误使我的hugo打不开，报错如下:""当前运行环境是“development”。“评论系统”、“PWA”、“CDN”和“fingerprint”不会启用。
WARN  Deprecated parameter detection until v0.3.13
  - The parameter `params.gitRepo` and `params.page.edit` is deprecated since v0.3.0, use `params.gitInfo` instead.
  - The parameter `params.ibruce` is deprecated since v0.3.9, use `params.busuanzi` instead.
WARN  found no layout file for "baiduurls" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN  found no layout file for "json" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
Built in 187 ms
Error: error building site: logged 1 error(s)
 "",大佬可以提供点建议给小白吗
”

**Reply by leegical** [reply]
距离这篇文章发布，Fixit主题已经更新了很多次。3的报错提示你了，不蒜子的参数变了。最新主题的搭建可以参考官方文档https://fixit.lruihao.cn/zh-cn/
